### ISSUE TITLE
Error while initializing WordNetCorpusReader

### ISSUE THREAD

**Issue by danmincu** [AlexMoreo/WNDomains#1]
When running main.py with Pyton 3.9 + NLTK 3.7 the following error pops up
```
  File "/usr/local/lib/python3.9/site-packages/nltk/corpus/reader/wordnet.py", line 1285, in omw_prov
    fileids = self._omw_reader.fileids()
AttributeError: 'str' object has no attribute 'fileids'
```
Any ideas what is happening?

**Comment by AlexMoreo** [comment]
Hi danmincu,
This looks like an error raised by the wordnet implementation in nltk (and not by my code). I am afraid I cannot give you any help since I have not updated mine, but will come back to you ASAP if I find the time to reproduce your error.

**Comment by danmincu** [comment]
yeah, the [WordNetCorpusReader](https://github.com/AlexMoreo/WNDomains/blob/7ec1ac4371d632d184e784dcda7c48011fee421d/WNDomains.py#L21) constructor changed, the second parameter is no longer a string. The WoldNet-2.0 you mention in instructions is not getting used so the question is how can one refer to that in your code rather than the one involved in the above error?
